eclass/elisp-common.eclass

```diff
@@ -711,11 +711,15 @@ elisp-site-regen() {
 	;; DO NOT EDIT THIS FILE
 
 	;;; Code:
+
 	EOF
-	# Use sed instead of cat here, since files may miss a trailing newline.
-	sed '$q' "${sflist[@]}" </dev/null >>"${T}"/site-gentoo.el || ret=$?
+	# Concatenate all site initialisation files.
+	# /^;/s/\s*-\*-.*// deletes file-local-var cookies from comment lines.
```

Comment:
this is a bit overzealous but if that's fine, LGTM

Reply:
Hm, we could instead:
```suggestion
	# 1{/^;/s/\s*-\*-.*-\*-//} deletes file-local-var cookies from line 1.
```